rust/perspective-viewer/src/rust/components/column_selector/active_column.rs

```diff
@@ -404,7 +404,7 @@ impl Component for ActiveColumn {
 
                             <div class="column-selector-column-border">
 
-                                <TypeIcon ty={col_type} />
+                                <TypeIcon absolute=true ty={col_type} />
```

Comment:
This is purely a style property, and if we can avoid expressing it in the Yew code, we should. In this case, `"absolute"` is a class name that sets a property in CSS already ... the `TypeIcon`s that use this property are all children of the `#settings_panel`, so this can be done in just that CSS rule.